### PR TITLE
Added latest Rc Smoothing changes

### PR DESCRIPTION
**Pull request by Asizon**
This PR goes with part of https://github.com/betaflight/betaflight/pull/10727 and now is 100% coordination finished with this PR changes.
Added Rc smoothing changes maintaining the compatibility and functions of the previous one. I have used some constants but im not sure were they need to go @McGiverGim . I also use text replaces for i18n, but we can think on create new html group and hide the old, what do you think guys?

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:536]:
```suggestion
        let smoothingOnOff = semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44) ? FC.RX_CONFIG.rcSmoothingMode : FC.RX_CONFIG.rcSmoothinType;
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:538]: Same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:540]: Same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:542]: Same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:565]:
```suggestion
            if (setPointCutoff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:582]:
```suggestion
            if (feedforwardCutoff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:884]: Same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:901]:
```suggestion
        if (feedforwardCutoff === 0 || setPointCutoff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:921]:
```suggestion
    if (smoothingOnOff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:939]:
```suggestion
    if (feedforwardCutoff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:942]:
```suggestion
    if (setPointCutoff === 0) {
```

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:886]: same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:888]: same as above

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:890]:
```suggestion
    let autoFactor = semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44) ? FC.RX_CONFIG.rcSmoothingAutoFactor : FX.RX_CONFIG.rcSmoothingAutoSmoothness;
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Please also solve the sonar complaints, that's why the Windows build is failing...

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
I prefer always to have all the elements in the HTML and hide the elements that don't need to be here. It seems to me that needs less maintenance that to have a text totally dynamic.
For the rest, it seems good to me.

File: src/js/tabs/receiver.js
Comment (McGiverGim) [inline src/js/tabs/receiver.js:911]: Esto debe ir al i18n?

**Comment by Asizon** [comment]
After some talks with @McGiverGim I have reanmed old wrong named features, also removed unneeded MSP changes. Basically 80% of the PR are a renames.

File: src/js/tabs/receiver.js
Comment (haslinghuis) [reply src/js/tabs/receiver.js:892]: ON and OFF should be translated.

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
